db/migrate/20230711155304_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :author, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Kindly include a text column, `timestamps`, and the necessary references for `author` and `post` in the `comments` table, adhering to naming conventions and ensuring the appropriate foreign key relationships are established. :pray: 

_note: By making those changes, the code for creating the `comments` table will adhere to naming conventions, establish proper foreign key relationships, and include necessary columns and timestamps, thereby ensuring improved functionality and data integrity. :wink:_

Reply:
It's there, check other migration files.

https://github.com/prius2055/rails-blogging-app/blob/760daceb066491b3fed63429c1a60cecf07cd262/db/migrate/20230711163627_add_user_ref_to_posts.rb#L1C1-L1C1

schema file
https://github.com/prius2055/rails-blogging-app/blob/760daceb066491b3fed63429c1a60cecf07cd262/db/schema.rb#L1